README.md

```diff
@@ -78,6 +78,18 @@ There are two methods by which you can connect your Mattermost account to your M
     - Click on that link. If it asks for login, enter the Microsoft credentials for the dummy account created following [these steps](./docs/azure_setup.md#step-2-create-a-user-account-to-act-as-a-bot).
     - Refer [here](./docs/azure_setup.md#step-2-create-a-user-account-to-act-as-a-bot) for more details.
 
-- **FAQs**
+## Slash commands
+
+- `/msteams-sync connect` :- This command is used to connect your Mattermost account to MS Teams account.
+- `/msteams-sync disconnect` :- This command is used to disconnect your Mattermost account from MS Teams account.
+- `/msteams-sync connect-bot` :- This command is used to connect the bot account in Mattermost to an account in MS Teams and it can only be run by system admins.
+- `/msteams-sync disconnect-bot` :- This command is used to disconnect the bot account in Mattermost from the MS Teams account and it can only be run by system admins.
+- `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who have roles channel admins and above. To run this command, you must have your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID. You can get both these values using the slash command autocomplete that shows up after you type or select the link command.
```

Comment:
```suggestion
- `/msteams-sync link` :- This command is used to link the currently active Mattermost channel with an MS Teams channel and it can only be run by users who are channel admins and above. To run this command, you must have your Mattermost account connected with MS Teams. This command takes two arguments - MS Teams team ID and channel ID which you can get from command autocomplete.
```